packages/rum/test/htmlAst.ts

```diff
@@ -32,8 +32,8 @@ export const generateLeanSerializedDoc = (htmlContent: string, privacyTag: strin
     serializeNodeWithId(newDoc, {
       document: newDoc,
       parentNodePrivacyLevel: NodePrivacyLevel.ALLOW,
-    })!
-  ) as SerializedNodeWithId
+    })! as unknown as Record<string, unknown>
+  ) as unknown as SerializedNodeWithId
```

Comment:
I agree, those types are far from perfect, but I don't think it should be changed in this PR since they were already there. Let's change that in a follow-up task

Reply:
Sounds good to me